test/addons-napi/test_buffer/test_buffer.c

```diff
@@ -12,7 +12,7 @@
 
 #define NAPI_CALL(env, theCall)                                           \
   if (theCall != napi_ok) {                                               \
-    const char *errorMessage = napi_get_last_error_info()->error_message; \
+    const char* errorMessage = napi_get_last_error_info()->error_message; \
```

Comment:
Yeah, I used clang-format to start, and then fixed the rest by hand. https://github.com/nodejs/abi-stable-node/pull/140#issuecomment-285838900 has details on the command line used